### DISCUSSION TITLE
无法加载 DLL“MediaInfo.dll”: 找不到指定的模块

### DISCUSSION THREAD

**Discussion by zaonie** [hooke007/mpv_PlayKit#32] [libmpv]
请问大佬们，这个问题咋解决：用MPV.NET-CM播放视频，弹出下列提示。

无法加载 DLL“MediaInfo.dll”: 找不到指定的模块。 (异常来自 HRESULT:0x8007007E)。

System.DllNotFoundException: 无法加载 DLL“MediaInfo.dll”: 找不到指定的模块。 (异常来自 HRESULT:0x8007007E)。
   在 MediaInfo.MediaInfo_New()
   在 MediaInfo..ctor(String file) 位置 C:\Users\hooke\Desktop\Magpie_v0.6.1\mpv.net_CM-cm_test\src\Native\MediaInfo.cs:行号 11
   在 mpvnet.CorePlayer.ReadMetaData() 位置 C:\Users\hooke\Desktop\Magpie_v0.6.1\mpv.net_CM-cm_test\src\Misc\CorePlayer.cs:行号 1384
   在 mpvnet.App.<>c__DisplayClass76_0.<RunTask>b__0() 位置 C:\Users\hooke\Desktop\Magpie_v0.6.1\mpv.net_CM-cm_test\src\Misc\App.cs:行号 126

**Comment by hooke007** [comment]
**MediaInfo.dll**
你把这个文件给删了？

如果你的目录里只有三个文件说明你只下了一个patch（补丁包）
release的页面有说明需要在5493基础上更新此补丁

**Reply by zaonie** [reply]
大佬好，我下了你的OTA5502，还去sourceforge下了libmpv的最新包：“mpv-dev-x86_64-20211010-git-b3f3c3f.7z”。它里面也没有MediaInfo.dll.
"mpv-dev-x86_64"里面只有3个文件+1个include文件夹。include文件夹里也没有MediaInfo.dll

**Reply by hooke007** [reply]
你应该看我release的说明而不是找无关文件
![Snipaste_2021-10-12_23-49-41](https://user-images.githubusercontent.com/41094733/136988664-3c0ddbb0-ffc4-4d87-a707-d6102088eb03.jpg)

**Reply by zaonie** [reply]
原来如此~~对不起，我马虎了。谢谢大佬的无私帮助~!
